pyprocessing/pyprocessing.py

```diff
@@ -20,14 +21,18 @@ class PPNamespace(dict):
         'fill': '#ffffff',
         'width': 640,
         'height': 480,
+        'window_offset': (2, 2),
+        'window_title': 'pyprocessing',
+        'window_resizable': False,
+        'window_icon': PImage('pyprocessing/images/icon16x16.png')
```

Comment:
have you tested that the path does not fail when installing (python setup.py install) and importing the package from a different location ?

Reply:
It works now, but I'm not sure if my solution is the best way to do it

Reply:
Ideally, PImage should be able to take in an image in base64 format, thus eliminating the problem altogether.

I'll see if I can make a commit that adds that feature, and fix that line of code then.